### PR TITLE
[orchestrator] Start devices with specific bootloader build

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: frontend/src/host_orchestrator/orchestrator/instancemanager.go [hunk 1]
```diff
@@ -235,7 +235,7 @@ func defaultMainBuild() *apiv1.AndroidCIBuild {
 
 func (m *CVDToolInstanceManager) launchFromAndroidCI(
 	req apiv1.CreateCVDRequest, op apiv1.Operation) (*apiv1.CVD, error) {
-	var mainBuild, kernelBuild *apiv1.AndroidCIBuild = defaultMainBuild(), nil
+	var mainBuild, kernelBuild, bootloaderBuild *apiv1.AndroidCIBuild = defaultMainBuild(), nil, nil
```
Comment (jemoreira) [inline frontend/src/host_orchestrator/orchestrator/instancemanager.go:238]: nit: please use 3 lines. With longer variable names the code is less readable and the relative savings in space and typing are minuscule.

File: frontend/src/host_orchestrator/orchestrator/instancemanager.go [hunk 2]
```diff
@@ -530,7 +557,10 @@ type startCVDParams struct {
 	InstanceNumber   uint32
 	MainArtifactsDir string
 	RuntimeDir       string
-	KernelDir        string
+	// OPTIONAL. If set, kernel relevant artifacts will be pulled this dir.
```
Comment (jemoreira) [inline frontend/src/host_orchestrator/orchestrator/instancemanager.go:560]: "from this dir"?

**Review by jemoreira — APPROVED** [review]

File: frontend/src/host_orchestrator/orchestrator/instancemanager.go (hunk 2 above)
Comment (ser-io) [reply frontend/src/host_orchestrator/orchestrator/instancemanager.go]: done

File: frontend/src/host_orchestrator/orchestrator/instancemanager.go (hunk 1 above)
Comment (ser-io) [reply frontend/src/host_orchestrator/orchestrator/instancemanager.go]: done

**Pull request merged**